encrypted-media/idlharness.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <title>Encrypted Media Extentions IDL test</title>
+    <link rel="help" href="https://w3c.github.io/encrypted-media/">
+
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+    <script src=/resources/WebIDLParser.js></script>
+    <script src=/resources/idlharness.js></script>
+    <script src=/encrypted-media/util/fetch.js></script>
+  </head>
+  <body>
+    <h1 class="instructions">Description</h1>
+    <p class="instructions">
+      This test verifies that the implementations of the Encrypted Media Extensions API matchs its WebIDL definition.
+    </p>
+
+    <div id='log'></div>
+
+    <script>
+        setup(function() {
+
+            fetch( 'EncryptedMediaExtensions.idl')
+            .then( function( response ) {
+                if ( !response.ok ) throw new Error( 'IDL fetch failed' );
+                return response.text();
+            } )
+            .then( function( idls ) {
+
+                var idl_array = new IdlArray();
+                idl_array.add_untested_idls("[PrimaryGlobal] interface Window {};");
+                idl_array.add_untested_idls("interface Navigator {};");
```

Comment:
Does this also exclude our partial interface additions? I'm sure this is documented somewhere, but a comment above line 33 might be helpful.